documents/api/viewsets.py

```diff
@@ -80,6 +80,33 @@ def retrieve(self, request, pk, *args, **kwargs):
             as_attachment=True,
         )
 
+    @login_required()
+    def destroy(self, request, *args, **kwargs):
+        attachment = self.get_object()
+
+        # The user is the owner of the document
+        is_owner = request.user == attachment.document.user
+
+        # Only owners are allowed to remove the attachments,
+        # staff users aren't.
+        if not is_owner:
```

Comment:
I wonder if it's confusing that this is different from the documents viewset where you can delete a document if you're staff.

Reply:
Yeah, I thought so too, but the API spec doesn't say anything for staff members so... 🤔 we could maybe keep it for consistency, but I can imagine that staff won't be using the functionality very often.

Reply:
It's the same for adding attachments to a document, only owners (or organization owners) are allowed to, but not staff members.

Reply:
Well, I guess this is good for now. :+1: We can always change it later if there's a need.